perllib/FixMyStreet/Cobrand/Default.pm

```diff
@@ -728,6 +728,9 @@ sub admin_pages {
     if ( $user->has_body_permission_to('emergency_message_edit') ) {
         $pages->{emergencymessage} = [ _('Emergency message'), 12 ];
     }
+    if ( $user->has_body_permission_to('wasteworks_config') ) {
+        $pages->{waste} = [ _('WasteWorks config'), 14];
```

Comment:
What does 14 refer to?

Reply:
The admin nav items are ordered by that value, so this puts it at the end:
<img width="910" alt="image" src="https://user-images.githubusercontent.com/4776/188166680-b462d0c7-0d0f-492d-a7fe-86f82b11a08f.png">